server.js

```diff
@@ -1,30 +1,99 @@
 import express from "express";
 import cors from "cors";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+import booksData from "./data/books.json";
+
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json(
+    {
+      "Open API": "Books",
+      "Routes": [
+        {
+          "/books": "Get all books",
+          "/books/:id": "Get book whose id matches",
+          "/ratings/:rating": "Get books whose rating starts with the number provided"
+        }
+      ],
+      "Queries": [
+        {
+          "/books?author=string": "Get all books whose author contains the string provided",
+          "/books?title=string": "Get all books whose title contains the string provided",
+        }
+      ]
+    }
+  );
+});
+
```

Comment:
Really neat with this overview on the landing page! All the paths works wonderfully and great that with author and title you can use different strings for the search and do not need to type the entire title/authors name.  ✨📚

A small suggestion is that for consistency the path to id could be defined as "/books/bookID/:id" or similair, and for ratings "/books/ratings/:rating:", and I wrote in my explanation like this : 
"/netflixShows/release_year/'year of release'": "Get array of shows from specific release year",
but this is a really small thing : )